daemons/gptp/common/ieee1588port.cpp

```diff
@@ -230,8 +230,11 @@ bool IEEE1588Port::init_port(int delay[4])
 void IEEE1588Port::startPDelay() {
 	if (automotive_profile) {
 		if (log_min_mean_pdelay_req_interval != PTPMessageSignalling::sigMsgInterval_NoSend) {
+			long long unsigned int waitTime;
+			waitTime = ((long long) (pow((double)2, log_min_mean_pdelay_req_interval) *  1000000000.0));
```

Comment:
The use of pow for the log base 2 calculation for the various timers is used in number places in the gPTP daemon. I'd like to keep them consistent. Perhaps a separate general change would be to pull this out in a separate utility function and update all occurance of this calculation to make use of it.